frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js

```diff
@@ -65,6 +66,7 @@ function GameStatistics({ finalScores, onNewGameRequested }) {
         <div className="table-item">{getManualSubstractedGoals(TeamID.Team_blue)}</div>
         <div className="table-item">Manually substracted goals</div>
         <div className="table-item">{getManualSubstractedGoals(TeamID.Team_white)}</div>
+        <NumberOfShots blue={statistics.blueShotsCount} white={statistics.whiteShotsCount} />
```

Comment:
`NumberOfShots` component gets `statistics`, not `blue` and `white`

Reply:
It has been fixed